utils/virtcontainers-setup.sh

```diff
@@ -59,7 +59,7 @@ cp -r ${busybox_bundle}/* ${pause_bundle}/
 echo -e "Move to ${tmpdir}/${virtcontainers_build_dir}"
 pushd ${tmpdir}/${virtcontainers_build_dir}
 echo "Clone cni"
-git clone https://github.com/containernetworking/cni.git
+git clone https://github.com/containernetworking/plugins.git
```

Comment:
The point is that this change *should* have broken our `master` builds, but somehow didn't.

I've re-triggered builds on `master` with Travis and sure enough they fail as a result of the upstream change.

If we make `virtcontainers-setup.sh` use the same version of that repo as we use for vendoring, we're immune from upstream changes and know that what we are testing with is in sync with what virtcontainers is using.